freanalysis_land/freanalysis_land/land.py

```diff
@@ -0,0 +1,243 @@
+import json
+from analysis_scripts import AnalysisScript
+import intake
+import matplotlib.pyplot as plt
+import cartopy
+import cartopy.crs as ccrs
+import datetime
+import pandas as pd
+import re
+import xarray as xr
+
+
+class LandAnalysisScript(AnalysisScript):
+    """Abstract base class for analysis scripts.  User-defined analysis scripts
```

Comment:
adjust description- `LandAnalysisScript` is no longer an abstract base-class, but one that inherits from it